dashboard/app/views/plc/enrollment_task_assignments/show.html.haml

```diff
@@ -0,0 +1,17 @@
+-@task = @plc_enrollment_task_assignment.plc_task
+
+%p#notice= notice
+
+%p
+  This is the task assignment completion page for #{@task.name}
+
+%p
+  Here is the subpage for actually completing the task #{@task.type.underscore[4..-1]}
+= render partial: 'plc/enrollment_task_assignments/complete/' + @task.type.underscore[4..-1] unless @task.type == 'Plc::Task'
```

Comment:
It's easier to see what's going on here as `@task.type.underscore.sub(/^plc\//, '')` -- but really this logic should live in the model.